tests/__init__.py

```diff
@@ -183,3 +183,10 @@ def is_valid_for_schema_version(purpose: Union[Any], sv: SchemaVersion) -> bool:
 
 def mksname(purpose: Union[Any], sv: SchemaVersion, f: OutputFormat) -> str:
     return f'{_get_purpose_as_str(purpose)}-{sv.to_version()}.{_SNAME_EXT[f]}'
+
+
+class DpTuple(Tuple[SchemaVersion, str]):
+    @property
+    def __name__(self) -> str:
+        test_data_file = self[1]
+        return splitext(basename(test_data_file))[0]
```

Comment:
i know the `basename` looks unique, but it would be better to have have the schema version in the test name, too ... just to be safe
```patch
- return splitext(basename(test_data_file))[0]
+ return f'{self[0].to_version()}-{splitext(basename(self[1]))[0]}'
```

Reply:
Good point. Updated.